pvsystemprofiler/tilt_azimuth_study.py

```diff
@@ -132,10 +162,13 @@ def run(self, delta_method=('cooper', 'spencer')):
 
         delta_method = np.atleast_1d(delta_method)
         # calculate hour angle
-        self.omega = calculate_omega(self.data_sampling, self.num_days, self.lon_input, self.day_of_year,
-                                     self.gmt_offset)
+        self.omega = calculate_omega(
+            self.day_of_year, self.data_sampling, self.lon_input, self.gmt_offset
+        )
         # fit daily signal of cos theta
-        self.scale_factor_costheta, self.costheta_fit = find_fit_costheta(self.data_matrix, self.clear_index)
+        self.scale_factor_costheta, self.costheta_fit = find_fit_costheta(
```

Comment:
@bmeyers Looks like this is missing the `doy` parameter, and the returned values are different now as well (although I'm not certain how that would change this function), based on the changes in [pvsystemprofiler/algorithms/performance_model_estimation.py](https://github.com/slacgismo/solar-data-tools/pull/91/files#diff-f58829f1c72480217b9bd0318c96c61a7f773e80405ad07512d0ae618351db43).